app/backend/src/couchers/config.py

```diff
@@ -11,6 +11,8 @@
 CONFIG_OPTIONS = [
     # Whether we're in dev mode
     ("DEV", bool),
+    # Whether we're fg (answering API queries) or bg processing
+    ("ROLE", ["fg", "bg", "both"], "both"),
```

Comment:
I would prefer a shell script starting the two services and backgrounding one of them with `&`, instead of all the boilerplate in Python to create two processes.

Reply:
Let's discuss in the meeting today.

It's kind of messy at the moment, I'll try to clean it up a tad bit.

The bottom line is that I'd like it to be as simple as possible for new devs to spin it up, and that means having only one backend container running. But we can have config to run them separately and should do that in prod.

Reply:
It is perfectly fine to have a shell script as an entry point in a container. See you tonight!

Reply:
The outcome of the meeting is that it doesn't really matter if we fork in Python or in shell so we could leave it as it stands. Resolving conversation.